src/shared/utils/exportData/parseResponseValue.ts

```diff
@@ -11,18 +12,27 @@ import {
   DecryptedSliderRowsAnswer,
   DecryptedStabilityTrackerAnswer,
   DecryptedTimeAnswer,
+  ExtendedEvent,
+  ExtendedExportAnswerWithoutEncryption,
 } from 'shared/types';
-import {
-  Item,
-  SingleAndMultipleSelectRowsResponseValues,
-  SliderRowsResponseValues,
-} from 'shared/state';
+import { SingleAndMultipleSelectRowsResponseValues, SliderRowsResponseValues } from 'shared/state';
 import { getStabilityTrackerCsvName } from 'shared/utils';
 
 import { joinWihComma } from '../joinWihComma';
 import { getAnswerValue } from '../getAnswerValue';
 
-export const parseResponseValue = (answer: AnswerDTO, activityItem: Item, id: string) => {
+export const parseResponseValue = <
+  T extends DecryptedAnswerData<ExtendedExportAnswerWithoutEncryption>,
+>(
+  item: T,
+  isEvent = false,
+) => {
+  let answer: AnswerDTO | undefined = item.answer;
```

Comment:
what about
const answer = isEvent ? (item ...).response : item.answer
to prevent using let?